index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Boolean-uk-html-twitter</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="middle-container">
+            <div class="left-c">
+                <div class="left-header">
+                    <div class="icons">
+                        <img src="/images/logo-blue.svg" alt="logo">
+                    </div>
+                </div>
+                <div class="container-nav">
+                    <div class="nav">
+                        <div class="left-icons">
+                            <img src="/images/explore-blue.svg" />
+                        </div>
+                        <div class="left-link">
+                            <ul>
+                                <li><a href="https://mobile.twitter.com/explore">Explore</a></li>
+                            </ul>
+                        </div>
+
+                    </div>
+                    <div class="nav">
+                        <div class="left-icons">
+                            <img src="/images/notifications-blue.svg" />
+                        </div>
+                        <div class="left-link">
+                            <ul>
+                                <li><a href="https://mobile.twitter.com/notifications">Notifications</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="menu">
+                <div class="profile">
+                    <div class="top-profile">
+                        <div class="back-icon">
+                            <div>Back</div>
+                        </div>
+                        <div class="profile-name">
+                            <div class="name">Lewis Hamilton</div>
+                            <div class="tweets">1,155 Tweets</div>
+                        </div>
+                    </div>
+                    <div class="profile-banner">
+                        <a href="https://twitter.com/LewisHamilton/header_photo"><img
+                                src="https://pbs.twimg.com/profile_banners/213969309/1618767914/1500x500"
+                                alt="Lewis Hamilton's Profile Banner" width="598px" height="199px" /></a>
+                    </div>
+                    <div class="profile-info">
+                        <div class="row-1">
+                            <div class="profile-pic">
+                                <a href="https://twitter.com/LewisHamilton/photo">
```

Comment:
<a> normally you want the linkt open in a seperate page/tab, to do this you add target="_blank" inside the tag